1. Basic Web Development/frontend/html/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
```

Comment:
You should rebase this branch over `main` as a lot of those files were added via your last PR, now you are just modifying them.